src/libs/PolicyUtils.js

```diff
@@ -96,6 +96,19 @@ function isExpensifyTeam(email) {
     return emailDomain === CONST.EXPENSIFY_PARTNER_NAME || emailDomain === CONST.EMAIL.GUIDES_DOMAIN;
 }
 
+/**
+ * Check if the policy member should be hidden
+ * If online and no errors and pending deletion then hide the member
+ * @param {Object} errors
+ * @param {String} pendingAction
+ * @param {Boolean} isOffline
+ * @returns {boolean}
+ */
+function shoulHideChildren(errors, pendingAction, isOffline) {
+    const hasErrors = !_.isEmpty(errors);
+    return !isOffline && pendingAction === 'delete' && !hasErrors;
+}
```

Comment:
@jayeshmangwani  Thanks for the initiative. I have a different opinion about this.

I think `shouldHideChildren` is not explain anything. Why do we need to hide the children? Whose children are we talking about? 😅

I also think `PolicyUtils` is not the right place. In `OfflineWithFeedback`, you can put any `errors` from any source, not just from the policy.

IMO we should keep it simple and just create a local method in `WorkspaceMembersPage` to handle `!props.network.isOffline && props.pendingAction === 'delete' && !hasErrors;`.

```js
isDeletedPolicyMember(policy) {
    return !this.props.network.isOffline && policy.pendingAction === 'delete' && _.isEmpty(policy.errors);
}
```

Reply:
@mollfpr Thanks for the suggestions, I have removed the unnecessary function from PolicyUtils and created a function on the page, and used there directly